tests/server/api.test.js

```diff
@@ -1,7 +1,7 @@
 const test = require('ava')
 const request = require('supertest')
 
-const app = require('../server/server')
+const app = require('../../server/server')
```

Comment:
I think the addition of the `server` folder is in another PR so these edits should be removed from this one.